test/readonly/models/tableSuffix/generated.rs

```diff
@@ -0,0 +1,55 @@
+/* @generated and managed by dsync */
+
+use crate::diesel::*;
+use crate::schema::*;
+use diesel::QueryResult;
+use serde::{Deserialize, Serialize};
+
+
+type ConnectionType = diesel::r2d2::PooledConnection<diesel::r2d2::ConnectionManager<diesel::PgConnection>>;
+
+#[derive(Debug, Serialize, Deserialize, Clone, Queryable, Insertable, AsChangeset, Selectable)]
+#[diesel(table_name=tableSuffix, primary_key(id))]
+pub struct TableSuffix {
+    pub id: crate::schema::sql_types::Int,
+    pub testprop: crate::schema::sql_types::Int,
```

Comment:
strange... since when did Int become `crate::schema::sql_types::Int`? Shouldn't this be `i32` or something similar?

Reply:
thanks for catching this, this is because i did not add a number (like 2 or 4) at the end, and there does not exist a mapping for plain "Int" (only `Int2/4`) so it has fallen back on custom types

Reply:
fixed with d8b811715d68bd79a16f6055ac4f3e7c322e58b2 (not just from this PR)

Reply:
Is `Int` a legitimate type for diesel? We might want to handle this case~

Reply:
at least from my quick skim of [sql_types](https://github.com/diesel-rs/diesel/blob/b40900e3e9485a546b071682809cead0c95a8d14/diesel/src/sql_types/mod.rs#L105) in diesel, there is no plain `Int` type, the closest is `Int2`, `Int4` and `Int8` in terms of name